config/locales/en.yml

```diff
@@ -1658,6 +1658,79 @@ en:
             allow_registration_without:
               "true": "Competitors will be allowed to register for events they're not qualified for (I'll remove them later)"
               "false": "Competitors MUST be qualified to register for an event"
+        #context: new registration system
+    registration_v2:
+      pdf:
+        #context: <0> indicates an html tag for internationalizing in React, see https://docs.worldcubeassociation.org/guides/translating_the_website.html#using-react
+        link: "As a <0>%{pdfLink}</0>"
+        download: "Download all of the competitions details."
+      list:
```

Comment:
This mixes "enumerated values that describe what a column means" with "user-friendly help text to display below/above tables". They should go in separate branches.